### PR TITLE
Added challenge question to installnginx.md

### PR DESCRIPTION
**Pull request by trevorrobertsjr**
Challenge question provides clues for the correct helm install syntax.
Workshop participant can either figure out the syntax themselves or
expand the hidden text for the correct command.

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/helm_root/helm_nginx/installnginx.md [hunk 1]
```diff
@@ -8,11 +8,15 @@ Installing the Bitnami standalone NGINX web server Chart involves us using the [
 
 When we install using Helm, we need to provide a deployment name, or a random one will be assigned to the deployment automatically.
 
-Let's provide **mywebserver** as the deployment name as we install NGINX:
+Now, use the **helm** utility to **install** the **bitnami/nginx** chart and specify the name **mywebserver** for the Kubernetes deployment:
```
Comment (dalbhanj) [inline content/helm_root/helm_nginx/installnginx.md]: I like the challenge question. Can you use format mentioned in Wiki for consistency? 
(https://github.com/aws-samples/eks-workshop/wiki/Workhop-testing)

File: content/helm_root/helm_nginx/installnginx.md [hunk 2]
```diff
@@ -8,11 +8,15 @@ Installing the Bitnami standalone NGINX web server Chart involves us using the [
...
 
+**NOTE:** Consult the helm documentation link above or run the command ```helm install --help``` command to figure out the syntax
+
+{{%expand "ONLY if you get stuck" %}}
 ```
 helm install --name mywebserver bitnami/nginx
 ```
+{{% /expand %}}
 
 Upon running this command, the output confirms the types of k8s objects that were created as a result:
```
Comment (dalbhanj) [inline content/helm_root/helm_nginx/installnginx.md]: I think this need to start with 'Once you run the command, ...'

Comment (trevorrobertsjr) [reply content/helm_root/helm_nginx/installnginx.md]: That is the original text from whoever wrote it. I will update it with your wording, though.

File: content/helm_root/helm_nginx/installnginx.md (hunk 1 above)
Comment (trevorrobertsjr) [reply content/helm_root/helm_nginx/installnginx.md]: I updated with a second commit to fix the challenge formatting. can you take another look?

File: content/helm_root/helm_nginx/installnginx.md (hunk 2 above)
Comment (trevorrobertsjr) [reply content/helm_root/helm_nginx/installnginx.md]: I updated this wording too in the second commit.

Comment (brentley) [reply content/helm_root/helm_nginx/installnginx.md]: It looks like this has been resolved.

File: content/helm_root/helm_nginx/installnginx.md (hunk 1 above)
Comment (brentley) [reply content/helm_root/helm_nginx/installnginx.md]: it looks like this has been resolved.

**Pull request merged**